synapse/http/server.py

```diff
@@ -383,41 +388,115 @@ def _get_handler_for_request(self, request):
         return _unrecognised_request_handler, "unrecognised_request_handler", {}
 
     def _send_response(
-        self, request, code, response_json_object, response_code_message=None
+        self, request, code, response_object,
     ):
+        """Implements _AsyncResource._send_response
+        """
         # TODO: Only enable CORS for the requests that need it.
         respond_with_json(
             request,
             code,
-            response_json_object,
+            response_object,
             send_cors=True,
-            response_code_message=response_code_message,
```

Comment:
I looked briefly and I suspect that this is unused everywhere? Might be worth checking as a follow-up.

Reply:
Oh yes, sorry, it is entirely unused.